In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
%matplotlib inline 
import seaborn as sns
import pickle

## 1) Abstract 

## 2) Introduction 

## 3) Research Questions 

## 4) EDA 

#### For more detailed EDA please read (insert EDA notebook link here)

#### Loading the Data Frame 

In [2]:
#Read in Data 
spotify_df = pd.read_pickle('../Data/spotify_df.pkl')

In [3]:
spotify_df.info()

<class 'pandas.core.frame.DataFrame'>
MultiIndex: 10901 entries, ('The Sound of Pop', 0) to ('The Sound of Jazz', 397)
Data columns (total 21 columns):
 #   Column            Non-Null Count  Dtype  
---  ------            --------------  -----  
 0   track_id          10901 non-null  object 
 1   artist            10901 non-null  object 
 2   artist_id         10901 non-null  object 
 3   album             10901 non-null  object 
 4   trackName         10901 non-null  object 
 5   acousticness      10901 non-null  float64
 6   danceability      10901 non-null  float64
 7   duration_ms       10901 non-null  int64  
 8   energy            10901 non-null  float64
 9   instrumentalness  10901 non-null  float64
 10  key               10901 non-null  int64  
 11  liveness          10901 non-null  float64
 12  loudness          10901 non-null  float64
 13  mode              10901 non-null  int64  
 14  speechiness       10901 non-null  float64
 15  tempo             10901 non-null  float64
 1

  - We have a data set of 10901 tracks and 21 different varaibles.
  - The data set contains no missing or Null values. 
  

## 5) Data Prep & Feature Engineering 

## 6) Modeling 

## 7) Model Results 

## 8) Conclusion 

## 9) Limitations and Future Work

## 10) References 